tests/e2e/upgradecluster/Vagrantfile

```diff
@@ -10,7 +10,7 @@ REGISTRY = (ENV['E2E_REGISTRY'] || "")
 GOCOVER = (ENV['E2E_GOCOVER'] || "")
 NODE_CPUS = (ENV['E2E_NODE_CPUS'] || 2).to_i
 NODE_MEMORY = (ENV['E2E_NODE_MEMORY'] || 2048).to_i
-GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "master")
+GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "v1.30")
```

Comment:
is there any way to template this from the CI environment, instead of having to explicitly set it on each branch?

Reply:
Yeah I don't love the manual setting. Probably get to that in the next few weeks.